egs/libri_css/asr1_mono/local/download_asr.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Copyright  2020  University of Stuttgart (Author: Pavel Denisov)
+# Apache 2.0
+
+# Begin configuration section.
+# End configuration section
+. ./utils/parse_options.sh  # accept options
+
+. ./path.sh
+
+echo >&2 "$0" "$@"
+if [ $# -ne 0 ] ; then
+  echo >&2 "$0" "$@"
+  echo >&2 "$0: Error: wrong number of arguments"
+  echo -e >&2 "Usage:\n  $0"
+  exit 1
+fi
+
+
+set -e -o pipefail
+
+mkdir -p downloads
+dir=$(mktemp -d -p downloads)
+
```

Comment:
Can you please add some comments on what is this ASR model (like how much data it was trained on, augmentation used etc.)

Reply:
Done.